nautobot/core/templates/generic/object_list.html

```diff
@@ -53,49 +53,43 @@
-                                {% if saved_view == current_saved_view %}
-                                    <i class="mdi mdi-view-compact-outline text-bold" aria-hidden="true"></i>
-                                    <strong>
-                                        {{saved_view.name}}
-                                    </strong>
-                                {% else %}
-                                    <i class="mdi mdi-view-compact-outline text-muted" aria-hidden="true"></i>
+                {% for saved_view in saved_views %}
+                    <li>
+                        <a href="{{ saved_view.get_absolute_url }}">
+                            {% if saved_view == current_saved_view %}
+                                <i class="mdi mdi-view-compact-outline text-bold" aria-hidden="true"></i>
+                                <strong>
                                     {{saved_view.name}}
-                                {% endif %}
-                            </a>
-                        </li>
-                    {% endfor %}
-                    {% if saved_views %}
-                        <li role="separator" class="divider"></li>
-                    {% endif %}
+                                </strong>
+                            {% else %}
+                                <i class="mdi mdi-view-compact-outline text-muted" aria-hidden="true"></i>
+                                {{saved_view.name}}
+                            {% endif %}
+                        </a>
+                    </li>
+                {% endfor %}
+                {% if saved_views %}
+                    <li role="separator" class="divider"></li>
                 {% endif %}
                 {% if current_saved_view %}
-                    {% if perms.users.change_savedview %}
-                        <li>
-                            <a href="{% url 'user:savedview_edit' pk=request.GET.saved_view %}{% if request.GET %}?{{ request.GET.urlencode }}{% endif %}">
-                                <i class="mdi mdi-content-save-outline text-muted" aria-hidden="true"></i>
-                                <span>Update Current View</span>
-                            </a>
-                        </li>
-                    {% endif %}
-                {% endif %}
-                {% if perms.users.add_savedview %}
                     <li>
-                        <a href="" data-toggle="modal" data-target="#saved_view_modal" title="Save Current View As">
-                            <i class="mdi mdi-content-save-plus-outline text-muted" aria-hidden="true"></i>
-                            <span>Save As New View</span>
+                        <a href="{% url 'user:savedview_edit' pk=request.GET.saved_view %}{% if request.GET %}?{{ request.GET.urlencode }}{% endif %}">
+                            <i class="mdi mdi-content-save-outline text-muted" aria-hidden="true"></i>
+                            <span>Update Current View</span>
```

Comment:
```suggestion
                            <span>Update Current Saved View</span>
```

Also I wonder if it'd make sense to move this below the "Save As New View" item so that we only need the one `if current_saved_view` block?

Reply:
Little pushback on that, I think the first operation you would look for is an update. So putting it first would make sense. The if...endif...block is not too much overhead.

Reply:
Fair enough!